src/StackHCI/help/Set-AzStackHCI.md

```diff
@@ -0,0 +1,281 @@
+---
+external help file: Az.StackHCI-help.xml
+Module Name: Az.StackHCI
+online version: https://docs.microsoft.com/powershell/module/az.stackhci/set-azstackhci
+schema: 2.0.0
+---
+
+# Set-AzStackHCI
+
+## SYNOPSIS
+Set-AzStackHCI modifies resource properties of the Microsoft.AzureStackHCI cloud resource representing the on-premises cluster to enable or disable features.
+
+## SYNTAX
+
+```
+Set-AzStackHCI [[-ComputerName] <String>] [-Credential <PSCredential>] [-ResourceId <String>]
+ [-EnableWSSubscription <Boolean>] [-DiagnosticLevel <DiagnosticLevel>] [-TenantId <String>]
+ [-ArmAccessToken <String>] [-GraphAccessToken <String>] [-AccountId <String>] [-EnvironmentName <String>]
+ [-UseDeviceAuthentication] [-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Set-AzStackHCI modifies resource properties of the Microsoft.AzureStackHCI cloud resource representing the on-premises cluster to enable or disable features.
+
+## EXAMPLES
+
+### EXAMPLE 1
+```powershell
+PS C:\\\> Set-AzStackHCI -EnableWSSubscription $true
```

Comment:
```suggestion
PS C:\> Set-AzStackHCI -EnableWSSubscription $true
```
I don't know if the three backslash are for escaping, but in a codeblock in markdown you don't need that. Simply `PS C:\>` will do.

Please update all occurance